packages/labs/src/components/tag-input/tagInput.tsx

```diff
@@ -288,7 +303,7 @@ export class TagInput extends AbstractComponent<ITagInputProps, ITagInputState>
 
     private handleInputChange = (event: React.ChangeEvent<HTMLInputElement>) => {
         this.setState({ activeIndex: NONE, inputValue: event.currentTarget.value });
-        Utils.safeInvoke(this.props.inputProps.onChange, event);
```

Comment:
actually, we should _also_ `safeInvoke` this prop: no need to stop supporting it. i'd expect to see two safeInvokes on the right.

Reply:
Ah makes sense, added this back in